web/src/admin/blueprints/BlueprintListPage.ts

```diff
@@ -126,7 +126,7 @@ export class BlueprintListPage extends TablePage<BlueprintInstance> {
     row(item: BlueprintInstance): TemplateResult[] {
         let description = undefined;
         const descKey = "blueprints.goauthentik.io/description";
-        if (Object.hasOwn(item.metadata?.labels, descKey)) {
+        if (Object.hasOwn(item.metadata?.labels ?? new Object(), descKey)) {
```

Comment:
```suggestion
        if (item.metadata && item.metadata.labels && Object.hasOwn(item.metadata?.labels, descKey)) {
```

I think this reads slightly better